Demo/DemoViewController.swift

```diff
@@ -10,20 +10,19 @@ import UIKit
 import PKHUD
 
 class DemoViewController: UIViewController {
-    
-    let hiddenTextField = UITextField(frame: CGRect.zero)
 
+    let hiddenTextField = UITextField(frame: CGRect.zero)
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         view.addSubview(hiddenTextField)
         HUD.registerForKeyboardNotifications()
-        
+
```

Comment:
I remove unnecessary trailing whitespaces because Xcode11 beta5 warns:

> Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)